pkg/logbridge/logbridge.go

```diff
@@ -100,8 +129,9 @@ func (lb *LogBridge) lossyCopy(r io.Reader, lines chan []byte) {
 		case lines <- line:
 		default:
 			droppedLines++
+			lb.droppedLineCount.Inc(1)
```

Comment:
the rate limit will cause the channel (buffer) to fill up which will cause us to fall into this case

I agree that this isn't a precise metric, but I think it's better than nothing?